app/namespace_test.go

```diff
@@ -70,53 +70,6 @@ var describeNamespaceResponseServer = &workflowservice.DescribeNamespaceResponse
 	},
 }
 
-func (s *cliAppSuite) TestNamespaceUpdate() {
-	resp := describeNamespaceResponseServer
-	s.frontendClient.EXPECT().DescribeNamespace(gomock.Any(), gomock.Any()).Return(resp, nil).Times(2)
-	s.frontendClient.EXPECT().UpdateNamespace(gomock.Any(), gomock.Any()).Return(nil, nil).Times(2)
-	err := s.app.Run([]string{"", "operator", "namespace", "update", cliTestNamespace})
-	s.Nil(err)
-	err = s.app.Run([]string{"", "operator", "namespace", "update", "--description", "another desc", "--email", "another@uber.com", "--retention", "1", cliTestNamespace})
-	s.Nil(err)
-}
-
-func (s *cliAppSuite) TestNamespaceUpdate_Data() {
```

Comment:
Re-created these as E2E tests. Removed unit tests as these do not really test much, even here the usage was wrong and it read `"true"` as the name of the namespace